internal/operator/ocs/validations.go

```diff
@@ -0,0 +1,188 @@
+package ocs
+
+import (
+	"encoding/json"
+	"math"
+
+	"github.com/docker/go-units"
+	"github.com/openshift/assisted-service/internal/host"
+	"github.com/openshift/assisted-service/models"
+	"github.com/sirupsen/logrus"
+)
+
+//Info is used to return the parameters required for OCS Validations
+type Info struct {
+	CountOfOcsDisks      int64
+	CanDeployOcs         bool
+	OcsMinimalDeployment bool
+	StorageCapacity      int64
+}
+
+type Config struct {
+	OCSMinimumCPUCount        int64 `envconfig:"OCS_MINIMUM_CPU_COUNT" default:"24"`
+	OCSMinimumRAMGB           int64 `envconfig:"OCS_MINIMUM_RAM_GB" default:"66"`
+	OCSRequiredDisk           int64 `envconfig:"OCS_MINIMUM_DISK" default:"3"`
+	OCSRequiredHosts          int64 `envconfig:"OCS_MINIMUM_HOST" default:"3"`
+	OCSRequiredCPUCount       int64 `envconfig:"OCS_REQUIRED_CPU_COUNT" default:"30"`
+	OCSRequiredRAMGB          int64 `envconfig:"OCS_REQUIRED_RAM_GB" default:"72"`
+	OCSRequiredMasterCPUCount int64 `envconfig:"OCS_REQUIRED_MASTER_CPU_COUNT" default:"16"`
+	OCSRequiredWorkerCPUCount int64 `envconfig:"OCS_REQUIRED_WORKER_CPU_COUNT" default:"10"`
+	OCSRequiredMasterRAMGB    int64 `envconfig:"OCS_REQUIRED_MASTER_RAM_GB" default:"64"`
+	OCSRequiredWorkerRAMGB    int64 `envconfig:"OCS_REQUIRED_WORKER_RAM_GB" default:"32"`
+}
+
+// ValidateOCSRequirements is used to validate min requirements of OCS
+func (c *Config) ValidateOCSRequirements(log logrus.FieldLogger, cluster *models.Cluster, h host.API) *Info {
```

Comment:
Why does `Config` doing the implementation and not `OCSValidator` or something like this.